zerver/actions/create_realm.py

```diff
@@ -254,25 +257,36 @@ def do_create_realm(
         # We use acting_user=None for setting the initial plan type.
         do_change_realm_plan_type(realm, Realm.PLAN_TYPE_LIMITED, acting_user=None)
 
-    admin_realm = get_realm(settings.SYSTEM_BOT_REALM)
-    sender = get_system_bot(settings.NOTIFICATION_BOT, admin_realm.id)
-    # Send a notification to the admin realm
-    signup_message = _("Signups enabled")
+    # Send a notification to the admin realm when a new organization registers.
+    if settings.CORPORATE_ENABLED:
+        admin_realm = get_realm(settings.SYSTEM_BOT_REALM)
+        sender = get_system_bot(settings.NOTIFICATION_BOT, admin_realm.id)
 
-    try:
-        signups_stream = get_signups_stream(admin_realm)
-        topic = realm.display_subdomain
+        support_url = get_support_url(realm)
+        organization_type = get_org_type_display_name(realm.org_type)
 
-        internal_send_stream_message(
-            sender,
-            signups_stream,
-            topic,
-            signup_message,
+        message = "[{name}]({support_link}) ([{subdomain}]({realm_link})). Organization type: {type}".format(
+            name=realm.name,
+            subdomain=realm.display_subdomain,
+            realm_link=realm.uri,
+            support_link=support_url,
+            type=organization_type,
         )
```

Comment:
We've generally been using f-strings for most string formatting; though maybe this line-wraps in a more ugly way with it?

Reply:
I could make this change as a clean up commit in the next step of this project which will be adding the notifications for organization plan type changes. This was something I didn't catch to fix again when I removed the translation of the string with `_("...")`.